sycl/source/handler.cpp

```diff
@@ -127,17 +127,21 @@ event handler::finalize() {
   {
     // A counter is not good enough since we can have the same accessor several
```

Comment:
probably should remove this comment as well, since the unordered_set is being removed.

Reply:
Good spot! I missed that one. Thank you!